.github/workflows/dart_ddc.yaml

```diff
@@ -12,8 +12,7 @@ on:
         type: string
 
 jobs:
-  test:
-    name: Test
+  test-dart-ddc:
```

Comment:
NOTE: in order to properly distinguish jobs within a given workflow run, the jobs must not have a name field set and must have a unique identifiable name.  Please see the comments added to log_cw_metric.dart for more information.